WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -177,7 +177,7 @@ open class QuickStartTourGuide: NSObject {
             return
         }
         if element != currentElement {
-            let blogDetailEvents: [QuickStartTourElement] = [.blogDetailNavigation, .checklist, .themes, .viewSite, .sharing]
```

Comment:
What's the reason for adding these extra events here?

I'm not sure if this is an issue, but makes a slight change to the behaviour of the pages tour:

* Start the Pages tour
* Tap into the Pages screen
* Before this change, if you tap back the notice would stay visible and you could tap back into Pages and the notice would still be there.
* With this change, the notice disappears if you tap back into Pages.

Reply:
Thanks for pointing this out @frosty ! I think I had misinterpreted the usage of these events, thinking that just tapping on the initial action should dismiss the notice (and the spotlight), while instead, in the case of pages, it's the + that dismisses the notice. This came by observing that when I tap on "Blog Posts", it would dismiss any notice and spotlight, including the notice on the FAB that pops up from "Grow your audience -> Publish a post".
So maybe I should revert these changes and then consider if the blog post needs changes, what do you think?

Reply:
Went ahead and did the change, we can adjust it further in future PRs